Source/AppAuthTV/AppAuthTV.h

```diff
@@ -1,7 +1,7 @@
 /*! @file AppAuthTV.h
```

Comment:
I'd move this file directly under source, similar to AppAuth.h and AppAuthCore.h.